src/github/exceptions/user-not-found.exception.ts

```diff
@@ -0,0 +1,10 @@
+import { HttpException, HttpStatus } from '@nestjs/common';
+
+export class UserNotFound extends HttpException {
```

Comment:
```suggestion
export class UserNotFoundException extends Exception {
```

- suffix the exceptions by "Exception" for better clarity.
- the `UserNotFoundException` is not related to HTTP, so don't use HTTP concepts such as 404 status code. a meaningful string message is sufficient